packages/api-catalog-service/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:14-alpine
+FROM node:16.14.2-alpine
```

No need to lock it down to a specific minor/patch version.

```suggestion
FROM node:16-alpine
```